docs/modules/ROOT/nav.adoc

```diff
@@ -3,7 +3,7 @@
 * xref:compatibility.adoc[Compatibility]
 * Usage
 ** xref:usage/basic_usage.adoc[Basic Usage]
-** xref:usage/auto_correct.adoc[Auto-correct]
+** xref:usage/autocorrect.adoc[Autocorrect]
```

Comment:
I'm not sure how to get around this but this will break links.

Reply:
https://docs.antora.org/antora/latest/page/page-aliases/

Reply:
Yep, we'll need an alias here or we should just push this renaming for later.